esp/bindings/http/platform/httptransport.cpp

```diff
@@ -798,10 +798,10 @@ void CHttpMessage::logSOAPMessage(const char* message, const char* prefix)
     return;
 }
 
+static const char* POST_METHOD_STR = "POST ";
```

Comment:
I do wonder what happens when these methods are called via HTTP GET with the parameters in the URL.  But that would be another JIRA.

Reply:
Could that happen for those 3 methods? Please let me know so that I will open another JIRA.